### DISCUSSION TITLE
Captcha verification on sign up is impossible

### DISCUSSION THREAD

**Discussion by StefanOctavian** [npm/feedback#982] [Website] [answered]
I was trying to sign up for an account on npm registry and the captcha asked me to find which one out of 6 photos of dice sums up to 14, 20 TIMES!! And at the end, it just said it was wrong. I even tried the audio challenge too, which wasn't that bad (except for the fact that it too asked me to do it 20 times), and just the same it told me it was wrong and when I tried to do it again it woudn't load at all and checking the Chrome dev tools I found about 120 network errors (http 429 Too Many Requests). So something is definitely broken regarding that Captcha validation.

**Comment by hans-engstad** [comment]
I have the same issue, for me it was only 10 times though. When trying to report the issue by contacting them, I also have to solve the same Captcha to submit the support-request...

**Reply by hans-engstad** [reply, marked as answer]
I eventually solved my issue by using my phone with cellular data to register an account, this time I got a different captcha that was much easier.

**Reply by StefanOctavian** [reply]
Yes! I solved it the same way. Another solution might be using a VPN set to USA, but sometimes it blocks the Captcha.

**Comment by eivl** [comment]
After 8 tries and failing, I wrote to support to get help, I got a captcha at the end of the form here as well, so I could not sent them a message either. 

I will try this one!

**Reply by eivl** [reply]
It worked making an account using mobile and the above description.
